File: Api/ClientApp/src/app/models/worker.ts
import Util from "../util/util";
import WorkerType from "./worker.type";

export default class Worker {
    id: string;
    name: string;
    employmentDate: Date;
    salaryBase: number;
    workerType: WorkerType;
    chief: string;
    constructor() {
        this.id = Util.getEmptyId();
        this.name = "";
        this.salaryBase = 1000;
        this.employmentDate = new Date();
    }
}

File: Service/Model/Worker.cs
using System;

namespace Service.Model
{
    // ReSharper disable once ClassWithVirtualMembersNeverInherited.Global
    public class Worker
    {
        public virtual Guid Id { get; set; }
        public virtual string Name { get; set; }
        public virtual DateTime EmploymentDate { get; set; }
        public virtual decimal SalaryBase { get; set; }
        public virtual WorkerType WorkerType { get; set; }
        public virtual Guid? Chief { get; set; }
    }
}